Stylesheet/css/variables.css

```diff
@@ -1,11 +1,13 @@
 :root {
-    --st-primary-color: #800000;
```

Comment:
Any reason why you are using the prefix --st in your variable names?

Reply:
Just to differentiate it from other class variables. For my own convenience